### ISSUE TITLE
Seeking clickable link upon running server

### ISSUE THREAD

**Issue by mroswell** [h-enk/doks#341]
## Description

`Web Server is available at //localhost:1313/ (bind address 0.0.0.0)`
Without the `http:` the link is not clickable on the Mac terminal.

### Steps to reproduce
` % npm run start` after cloning repo 
 
### Expected result
`Web Server is available at http://localhost:1313/ (bind address 0.0.0.0)`
The `http:` enables the link to be clickable from the terminal. 

I am able to get this result when I set: `baseurl = "http://example.com"`

### Actual result
no http: so the link is not clickable. 

### Environment

```

> doks-child-theme@0.2.3 precheck
> npm version

{
  'doks-child-theme': '0.2.3',
  npm: '7.7.6',
  node: '15.14.0',
  v8: '8.6.395.17-node.28',
  uv: '1.41.0',
  zlib: '1.2.11',
  brotli: '1.0.9',
  ares: '1.17.1',
  modules: '88',
  nghttp2: '1.42.0',
  napi: '8',
  llhttp: '2.1.3',
  openssl: '1.1.1k+quic',
  cldr: '38.1',
  icu: '68.2',
  tz: '2020d',
  unicode: '13.0',
  ngtcp2: '0.1.0-DEV',
  nghttp3: '0.1.0-DEV'
}

> doks-child-theme@0.2.3 check
> hugo version

hugo v0.82.1-60618210+extended darwin/amd64 BuildDate=2021-04-20T11:02:50Z VendorInfo=gohugoio
```

**Comment by h-enk** [comment]
Hey @mroswell, thanks again — I will update this w/ Doks v0.3. For now:

In `package.json` update the start script to:

```
"start": "hugo server  -b=\"http://localhost:1313/\" --bind=0.0.0.0 --disableFastRender",
```

That should work.

**Comment by h-enk** [comment]
On second thought, I won't update the start script. I'd like to keep one place only (less confusion that way) for updating the development baseurl, namely in `./config/_default/config.toml`. Set `baseurl = "/"` (default, recommended), or `baseurl = "http://localhost:1313/"` (if you like).

Apart from that, I think it s/b fixed on terminal side (also applies to PowerShell btw). Note also that, `//localhost:1313/` is clickable when e.g. viewing the page source in Chrome + Firefox.

**Comment by h-enk** [comment]
Fixed w/ Doks [v0.3.4](https://github.com/h-enk/doks/releases/tag/v0.3.4)
